v0.0.1/model-schema.json

```diff
@@ -0,0 +1,192 @@
+      "description": "Public facing website for the model",
+      "type": "string",
+      "format": "uri"
+    },
+    "repo_url": {
+      "description": "Repository containing code for the model",
+      "type": "string",
+      "format": "uri"
+    },
+    "license": {
+      "description": "License for use of model output data",
+      "type": "string",
+      "enum": [
+        "apache-2.0",
+        "cc-by-4.0",
+        "cc-by-nc-4.0",
+        "cc-by-nc-nd-4.0",
+        "cc-by-sa-4.0",
+        "gpl-3.0",
+        "lgpl-3.0",
+        "mit"
+      ]
+    },
+    "include_viz": {
+      "description": "Indicator for whether the model should be included in the Hub’s visualization",
+      "type": "boolean"
+    },
+    "include_ensemble": {
+      "description": "Indicator for whether the model should be included in the Hub’s ensemble",
+      "type": "boolean"
+    },
+    "include_eval": {
+      "description": "Indicator for whether the model should be scored for inclusion in the Hub’s evaluations",
+      "type": "boolean"
+    },
+    "citation": {
+      "type": "string"
+    },
+    "team_funding": {
+      "type": "string"
+    },
+    "model_details": {
+      "type": "object",
+      "properties": {
+        "data_inputs": {
+          "type": "string"
+        },
+        "methods": {
+          "type": "string",
+          "maxLength": 200
+        },
+        "methods_long": {
+          "type": "string"
+        }
+      },
+      "additionalProperties": false,
+      "required": ["data_inputs", "methods"]
+    },
+    "ensemble_of_hub_models": {
+      "description": "Indicator for whether this model is an ensemble of other Hub models",
+      "type": "boolean"
+    },
+    "modeling_NPI": {
+      "type": "string"
+    },
+    "compliance_NPI": {
+      "type": "string"
+    },
+    "contact_tracing": {
+      "type": "string"
+    },
+    "testing": {
+      "type": "string"
+    },
+    "vaccine_efficacy_transmission": {
+      "type": "string"
+    },
+    "vaccine_efficacy_delay": {
+      "type": "string"
+    },
+    "vaccine_hesitancy": {
+      "type": "string"
+    },
+    "vaccine_immunity_duration": {
+      "type": "string"
+    },
+    "natural_immunity_duration": {
+      "type": "string"
+    },
+    "case_fatality_rate": {
+      "type": "string"
+    },
+    "infection_fatality_rate": {
+      "type": "string"
+    },
+    "asymptomatics": {
+      "type": "string"
+    },
+    "age_groups": {
+      "type": "string"
+    },
+    "importations": {
+      "type": "string"
+    },
+    "confidence_interval_method": {
+      "type": "string"
+    },
+    "calibration": {
+      "type": "string"
+    },
+    "spatial_structure": {
+      "type": "string[bc]"
+    }
+  },
+  "additionalProperties": false,
+  "required": [
+    "schema_version",
+    "team_name",
+    "team_abbr",
+    "model_name",
+    "model_abbr",
+    "model_contributors",
+    "website_url",
+    "license",
+    "include_viz",
```

Comment:
I suggest removing include_viz, include_ensemble, and include_eval, from the required fields as these seem like they might not be relevant for certain hubs.

Reply:
Am I correct in thinking that these properties could be added as required within a certain hub?

Reply:
I agree that not all hubs would use `include_viz`, etc., so they shouldn't be required for all hubs.

If I understand the current layout of this file correctly (maybe I'm missing something), it is directly specifying a schema for the model metadata files, not a second-order schema for what the schema for the model metadata files should look like.  That means that with the current organization, it might not be possible to add these properties as required within a certain hub?  More commentary [here](https://github.com/Infectious-Disease-Modeling-Hubs/schemas/pull/28#issuecomment-1371221107).